dali/pipeline/operator/sequence_shape_test.cc

```diff
@@ -148,6 +148,7 @@ class SequenceShapeUnfoldTest : public ::testing::Test {
     Container batch;
     constexpr bool is_device = std::is_same_v<batch_backend_t<Container>, GPUBackend>;
     batch.set_order(is_device ? AccessOrder(cuda_stream) : AccessOrder::host());
+    batch.SetBatchState(BatchState::Contiguous);  // TODO(klecki): or BatchState::Noncontiguous
```

Comment:
Unfolding does not impact contigioutiy.

Reply:
I expanded the tests to cover both variants.